railties/test/application/rake_test.rb

```diff
@@ -113,10 +113,8 @@ def test_should_not_eager_load_model_for_rake
         raise 'should not be pre-required for rake even eager_load=true'
       RUBY
 
-      Dir.chdir(app_path) do
-        assert system("bin/rails do_nothing RAILS_ENV=production"),
-               "should not be pre-required for rake even eager_load=true"
-      end
+      assert rails("do_nothing" "RAILS_ENV=production"),
```

Comment:
Asserting on a value that's always true isn't very useful.

.. and this is missing a comma, so it's also not running the command it intends to.

Reply:
@matthewd Should I just leave `rails("do_nothing", "RAILS_ENV=production")` without `assert`?

Reply:
Hmm.. maybe we should make `do_nothing` do a puts, then assert on that. That way it would've caught the missing comma.

Reply:
> Hmm.. maybe we should make do_nothing do a puts, then assert on that. That way it would've caught the missing comma.

Good idea! Done.